### PR TITLE
PRO-1180 parked properties cannot be edited

### PR DESCRIPTION
**Pull request by boutell**

### CODE REVIEW

File: modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue [hunk 1]
```diff
@@ -188,6 +188,9 @@ export default {
       const groupSet = {};
 
       this.schema.forEach(field => {
+        if (!this.filterOutParkedFields([ field.name ]).length) {
```
Comment (abea) [inline modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue:191]: This removes the field. Shouldn't we disable it instead?

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,13 @@
 * Four permissions roles are supported and enforced: guest, contributor, editor and admin. See the documentation for details. Pre-existing alpha users are automatically migrated to the admin role, as they already could do anything.
 * The admin bar menu is fully responsive to user roles.
 * The context bar entirely appears or disappears based on user roles.
+* Contributors can submit drafts for review ("Propose Changes" versus "Publish Changes").
```
Comment (abea) [inline CHANGELOG.md:11]: Are these unrelated lines in here from some inadvertent branching?

Comment (boutell) [reply CHANGELOG.md:11]: What unrelated lines? I only see one "+" here

Comment (boutell) [reply CHANGELOG.md:11]: They are all accurate to what is currently in 3.0 (apart from the new one)

**Review by abea — APPROVED** [review]
